runtime/util/ConfigurationPrinter.cpp

```diff
@@ -327,11 +325,13 @@ extern "C" void printMatchResult(
         fprintf(subject, "%s\n", strSubjectSort.c_str());
       }
       fflush(subject);
-      kllvm::printKORE(os, definitionPath, subjectFilename, false, true);
+      kllvm::printKORE(
+          os, definitionPath, subject_raii.getFilename(), false, true);
       os << "does not match pattern: \n";
       fprintf(pattern, "%s\n", matchLog[i].pattern);
       fflush(pattern);
-      kllvm::printKORE(os, definitionPath, patternFilename, false, true);
+      kllvm::printKORE(
+          os, definitionPath, pattern_raii.getFilename(), false, true);
```

Comment:
Can we clean this code up so we're using a consistent style for working with these files, rather than mixing C and C++-style code?